CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## CHANGELOG FOR 1.1.x
 #### 1.1.0
+- bugfix [#1504](https://github.com/ergonode/backend/issues/1504) Possibility of adding product itself as product relation (wiewiurdp)
```

Comment:
The bug was not yet introduced to released version therefore there should not be changelog entry for it.